.github/CODEOWNERS

```diff
@@ -37,8 +37,8 @@ exporter/sentryexporter/                             @open-telemetry/collector-c
 exporter/signalfxexporter/                           @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
 exporter/signalfxcorrelationexporter/                @open-telemetry/collector-contrib-approvers @jrcamp @keitwb
 exporter/splunkhecexporter/                          @open-telemetry/collector-contrib-approvers @atoulme
-exporter/stackdriverexporter/                        @open-telemetry/collector-contrib-approvers @dashpole
-exporter/googlecloudexporter/                        @open-telemetry/collector-contrib-approvers @dashpole
+exporter/stackdriverexporter/                        @open-telemetry/collector-contrib-approvers @GoogleCloudPlatform/open-telemetry
+exporter/googlecloudexporter/                        @open-telemetry/collector-contrib-approvers @GoogleCloudPlatform/open-telemetry
```

Comment:
I understand this, but not sure if github supports cross projects teams

Reply:
You are correct unfortunately: https://github.community/t/question-add-team-for-codeowners-from-other-organization/3069. I will revise this PR.